Manifests/ManagedPreferencesApplications/com.vpntracker.365mac-config.plist

```diff
@@ -234,6 +234,6 @@ A profile can consist of payloads with different version numbers. For example, c
 	<key>pfm_unique</key>
 	<true/>
 	<key>pfm_version</key>
-	<integer>1</integer>
+	<integer>2</integer>
```

Comment:
The version number can remain 1 (by convention it is only increased when preference keys are added or removed).

Reply:
Good to know. Fixed.